### PR TITLE
timestamp correctly showing

### PR DESCRIPTION
**Pull request by laurent-bouhnik-gh**

### CODE REVIEW

File: haystak.js [hunk 1]
```diff
@@ -12,12 +12,13 @@ var listHistory = function (list) {
    var questionValue = obj.question;
    var highlightValue = obj.highlight;
    var tagValue = obj.tags;
-   var timeValue = obj.date;
-   var questionAnswerPair = "<li>" + "<div class='content-container'>" + "<a href='" + keyTitle + "' target='_blank'>" + keyTitle + "</a>" + "<br><div class='para-header'>Q:</div>" + "<div class='para-content'>" + questionValue + "</div>" + "<div class='para-header'>A:</div>" + "<div class='para-content'>" + highlightValue +  "</div>" + "<div class='para-header'>Tags:</div>" + "<div class='para-content-tags'>" + tagValue + "</div>"  + "<div class='para-header'>Created :</div>" + "<div class='para-content-time'>" + timeValue + "</div>><br><br></li><br>";
+   var timeValue = ((obj.date).substring(0, 10));
+  //  var time =(timeValue).clearTime();
+  console.log((obj.date).substring(0, 10))  ;
```
Comment (fmlharrison) [inline haystak.js]: nice work!! Please can you remove the console.logs in the code...? ❤️

**Review by fmlharrison — COMMENTED** [review]
Just added a comment... great code! We need to do a code wide refactor!!

File: haystak.js (hunk 1 above)
Comment (laurent-bouhnik-gh) [reply haystak.js]: hahaha I saw it and thought of you. All done :)

**Review by fmlharrison — APPROVED** [review]
great!!!!

**Pull request closed without merging**